sbt-scala-native-bindgen/src/sbt-test/bindgen/generate/build.sbt

```diff
@@ -11,10 +11,14 @@ nativeBindgenPath := Some(file(System.getProperty("bindgen.path")))
 inConfig(Compile)(
   Def.settings(
     nativeBindings += {
+      //#sbt-exclude-prefix
       NativeBinding(resourceDirectory.value / "stdlib.h")
+      //#sbt-exclude-prefix
         .name("stdlib")
         .packageName("org.example.app.stdlib")
+        //#sbt-exclude-prefix
         .excludePrefix("__")
```

Comment:
I noticed that rendered snippet does not have indent on second line:
```sbt
NativeBinding(resourceDirectory.value / "stdlib.h")
.excludePrefix("__")
```
It can be fixed by changing comment indent:
```sbt
        .packageName("org.example.app.stdlib")
       //#sbt-exclude-prefix
         .excludePrefix("__")
```

Reply:
this also applies to [build.sbt:173](https://github.com/scala-native/scala-native-bindgen/pull/182/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR173)